### PR TITLE
Adding url decode

### PR DESCRIPTION
**Pull request by esharf**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/22581

## Description
Adding URL decoding for **hybrid-analysis-quick-scan-url** command.

## Does it break backward compatibility?
   - [ ] Yes
   - [x] No

## Must have
- [ ] Documentation

### CODE REVIEW

File: TestPlaybooks/playbook-HybridAnalysis-Test.yml [hunk 1]
```diff
@@ -891,14 +940,81 @@ tasks:
...
+    quietmode: 0
+  "31":
+    id: "31"
+    taskid: 03af2856-166f-4f3d-8718-c281393f1c38
+    type: regular
+    task:
+      elasticcommonfields: {}
+      id: 03af2856-166f-4f3d-8718-c281393f1c38
+      version: -1
+      name: Untitled Task
```
Comment (yaakovi) [inline TestPlaybooks/playbook-HybridAnalysis-Test.yml]: update task name

File: Integrations/integration-HybridAnalysis.yml [hunk 2]
```diff
@@ -630,6 +630,9 @@ script:
 
     //Quick Scan commands
     function quickScanUrl(scan_type, url, endpoint) {
+        url = url.replace('?', '%3F');
+        url = url.replace('=', '%3D');
+        url = url.replace('&', '%26');
         var body = 'scan_type=' + scan_type + '&url=' + url;
```
Comment (yaakovi) [inline Integrations/integration-HybridAnalysis.yml]: should `scan type` be encoded too?
```suggestion
        var body = 'scan_type=' + scan_type + '&url=' + encodeURIComponent(url);
```

File: Integrations/integration-HybridAnalysis_CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+Adding URL decoding for **hybrid-analysis-quick-scan-url** command
```
Comment (yaakovi) [inline Integrations/integration-HybridAnalysis_CHANGELOG.md]:
```suggestion
Adding URL decoding for **hybrid-analysis-quick-scan-url** command.
```

**Comment by kirbles19** [comment]
@esharf Done.

**Pull request merged**